.github/workflows/rnmobile-android-runner.yml

```diff
@@ -47,7 +47,7 @@ jobs:
               run: npm run native test:e2e:setup
 
             - name: Gradle cache
-              uses: gradle/gradle-build-action@3b1b3b9a2104c2b47fbae53f3938079c00c9bb87 # v3.0.0
+              uses: gradle/actions/setup-gradle@3b1b3b9a2104c2b47fbae53f3938079c00c9bb87 # v3.0.0
```

Comment:
This is currently failing due to the commit hash I believe 🤔 since it references the `gradle/gradle-build-action` [repo](https://github.com/gradle/gradle-build-action/commit/3b1b3b9a2104c2b47fbae53f3938079c00c9bb87).

I think we should change it to [ec92e829475ac0c2315ea8f9eced72db85bb337a](https://github.com/gradle/actions/commit/ec92e829475ac0c2315ea8f9eced72db85bb337a) which is the `v3.0.0` release in the `gradle/actions` repo.